include/uxr/agent/client/ProxyClient.hpp

```diff
@@ -63,6 +65,7 @@ class ProxyClient
 
 private:
     dds::xrce::CLIENT_Representation representation_;
+    Middleware* middleware_;
```

Comment:
I can change `Middleware* middleware_` to `Middleware& middleware_` but not yo `const Middleware& middleware_` because it is modified when create and delete functions are called.

Reply:
Done! Please check pull request  #56.